src/dask_nested/backends.py

```diff
@@ -0,0 +1,40 @@
+# Python 3.9 doesn't support "|" for types
+from __future__ import annotations
+
+import nested_pandas as npd
+import pandas as pd
+from dask.dataframe.backends import meta_nonempty_dataframe
+from dask.dataframe.dispatch import make_meta_dispatch
+from dask.dataframe.extensions import make_array_nonempty
+from dask.dataframe.utils import meta_nonempty
+from dask_expr import get_collection_type
+from nested_pandas.series.ext_array import NestedExtensionArray
+
+from .core import NestedFrame
+
+get_collection_type.register(npd.NestedFrame, lambda _: NestedFrame)
+
+# The following dispatch functions are defined as per the Dask extension guide:
+# https://docs.dask.org/en/latest/dataframe-extend.html
+
+
+@make_meta_dispatch.register(npd.NestedFrame)
+def make_meta_frame(x, index=None) -> npd.NestedFrame:
+    """Create an empty NestedFrame to use as Dask's underlying object meta."""
+    result = x.head(0)
+    return result
```

Comment:
I'm not sure what it is for, but should it be lazy?

Reply:
Yeah this was a bit of a placeholder, implemented a lazy way to do this